cmd/forch

```diff
@@ -6,5 +6,9 @@ else
     SOCK=inst/faucet/daq-faucet-$1/faucet_event.sock
 fi
 
+if [ ! -z "$2" ]; then
+    CPN_FILE="$2"
```

Comment:
Why does this conceptually need to be a separate file?  If there's one faucet setup for the network, keep it simple and derive the file based off of that (as per SOCK).

Reply:
this is the switches.yaml file

Reply:
Sorry, I meant "why does this need to be a separate command line argument?" -- why can't you just use the if-statement above and set CPN_FILE in parallel with SOCK?